specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/examples/ManagedInstanceListByResourceGroup.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters" : {
+    "subscriptionId": "20D7082A-0FC7-4468-82BD-542694D5042B",
+    "resourceGroupName" : "Test1",
+    "api-version" : "2015-05-01-preview"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "value" : [{
+            "location" : "onebox",
+            "id" : "/subscriptions/20d7082a-0fc7-4468-82bd-542694d5042b/resourceGroups/Test1/providers/Microsoft.Sql/managedInstances/testcl",
+            "name" : "testcl",
+            "type": "Microsoft.Sql/managedInstances",
+            "sku" : {
+            "name" : "CLS3",
+            "tier" : "Standard",
+            "capacity" : 1,
+            "family":"Gen5"
+            },
+            "properties" : {
+              "fullyQualifiedDomainName" : "testcl.database.windows.net",
+              "administratorLogin" : "cloudSA",
+              "state": "Ready",
+			  "vCores": 8,
+              "storageSizeInGB": 32,
+              "licenseType": "Full"
+            }
+          }, {
+            "location" : "onebox",
+            "id" : "/subscriptions/20d7082a-0fc7-4468-82bd-542694d5042b/resourceGroups/Test1/providers/Microsoft.Sql/managedInstances/testcl2",
+            "name" : "testcl2",
+            "type": "Microsoft.Sql/managedInstances",
+            "sku" : {
+              "name" : "CLS15",
+              "tier" : "Standard",
+              "capacity": 22,
+              "vCores": 24,
```

Comment:
vCores, storageSizeInGB and licenseType should be in "properties".

Reply:
Yes, sorry. We caught this and got the change checked into dsmain. Will update when we've hit production and are ready for check-in. Thank you.

Reply:
Is this just a swagger generating issue, or is it a production service issue?